From 38c5790d1b7e3d6c0e964501e2f01bc17fa6cf61 Mon Sep 17 00:00:00 2001
From: Juho Kuisma <kuisma.juho+github@gmail.com>
Date: Tue, 9 Jan 2024 14:09:45 +0200
Subject: [PATCH] Do not automatically check for updates

This makes startup faster and stops nagging about each new commit on
oh-my-zsh/oh-my-zsh master branch.
---
 oh-my-zsh.sh | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/oh-my-zsh.sh b/oh-my-zsh.sh
index 137ca3b..43436dc 100644
--- a/oh-my-zsh.sh
+++ b/oh-my-zsh.sh
@@ -65,8 +65,9 @@ fi
 mkdir -p "$ZSH_CACHE_DIR/completions"
 (( ${fpath[(Ie)"$ZSH_CACHE_DIR/completions"]} )) || fpath=("$ZSH_CACHE_DIR/completions" $fpath)
 
-# Check for updates on initial load...
-source "$ZSH/tools/check_for_upgrade.sh"
+# Do not check for updates automatically.
+# Uncomment to check and prompt for updates on each initial load.
+# source "$ZSH/tools/check_for_upgrade.sh"
 
 # Initializes Oh My Zsh
 
-- 
2.34.1

